app.js

```diff
@@ -8,3 +8,140 @@ open.addEventListener('click', () => {
+  github: 'https://www.geeksforgeeks.org/how-to-create-an-html-button-that-acts-like-a-link/',
+  live: '#',
+},
+];
+
+const projectsContainer = document.getElementById('works');
+const popup = document.querySelector('#popup');
+const popupTitle = popup.querySelector('#popupTitle');
+const popupClient = popup.querySelector('#popupClient');
+const popupRole = popup.querySelector('#popupRole');
+const popupYear = popup.querySelector('#popupYear');
+const popupImg = popup.querySelector('#popupImg');
+const popupDesc = popup.querySelector('#popupDesc');
+const popupTechs = popup.querySelector('#popupTechs');
+const popupLive = popup.querySelector('#popupLive');
+const popupSource = popup.querySelector('#popupSource');
+
+function displayPopup(project) {
+  const popUpContainer = document.getElementById('popUpContainer');
+  popUpContainer.classList.remove('display-none');
+  popupTitle.textContent = project.title;
+  popupImg.src = project.img;
+  popupClient.textContent = project.client;
+  popupRole.textContent = project.role;
+  popupYear.textContent = project.year;
+  popupDesc.textContent = project.desc;
+  popupTechs.innerHTML = '';
+  for (let i = 0; i < project.tech.length; i += 1) {
+    const li = document.createElement('li');
+    li.className = 'technology';
+    li.textContent = project.tech[i];
+    popupTechs.appendChild(li);
+  }
+  popupLive.addEventListener('click', () => {
+    window.location.href = project.live;
+  });
+  popupSource.addEventListener('click', () => {
+    window.location.href = project.github;
+  });
+
+  const exitPopUp = document.getElementById('exitPopUp');
+
+  exitPopUp.addEventListener('click', () => {
+    popUpContainer.classList.add('display-none');
+  });
+}
+
+for (let i = 0; i < projects.length; i += 1) {
+  const project = document.createElement('article');
+  project.className = 'project';
+  project.innerHTML = `
+    <img class="project__img" src = ${projects[i].img} alt = "frontpage first project">
+        <div>
+          <h3 class="project__title">${projects[i].title}</h3>
+          <p class="project__reference"><span class="project__client">${projects[i].client}</span> <svg width="8" height="8"
+            viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+          </svg> <span class="project__role">${projects[i].role}</span> <svg width="8" height="8" viewBox="0 0 8 8"
+            fill="none" xmlns="http://www.w3.org/2000/svg">
+              <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+            </svg> <span class="project__year">${projects[i].year}</span>
+          </p>
+          <p class="project__paragraph">${projects[i].desc}</p>
+          <ul class="project__technologies">
+            <li>${projects[i].tech[0]}</li>
+            <li>${projects[i].tech[1]}</li>
+            <li>${projects[i].tech[2]}</li>
+          </ul>
+          <button class="project__button">${projects[i].button}</button>
```

Comment:
- [ Optional] On the first render, users have to scroll down to see the buttons. this does not give a better user experience, and I strongly suggest that you fix this good job so far 👍 

# View 
![image](https://user-images.githubusercontent.com/50990925/216731512-3020c1e1-99fd-4643-b657-bdff56bbfe55.png)

Reply:
Hello @Shedrack-Sunday! For sure, thanks a lot for the feedback, I'm going to work on it!

Reply:
You're welcome